src/views/studio/l10n.json

```diff
@@ -87,6 +87,8 @@
     "studio.activityBecomeCurator": "{newCuratorProfileLink} accepted an invitation from {inviterProfileLink} to curate this studio",
     "studio.activityRemoveCurator": "{removerProfileLink} removed the curator {removedProfileLink}",
     "studio.activityBecomeOwner": "{promotedProfileLink} was promoted to manager by {promotorProfileLink}",
+    "studio.activityBecomeHost": "{newHostProfileLink} was made the studio host by {actorProfileLink}",
+    "studio.activityBecomeHostAdminActor": "{newHostProfileLink} was made the studio host by a Scratch Team member.",
```

Comment:
This is the only message with a period at the end - maybe fix for consistency?

Reply:
Good catch!